commerce-payment-web/src/main/resources/META-INF/resources/payment_method_action.jsp

```diff
@@ -22,8 +22,25 @@ CommercePaymentMethodGroupRelsDisplayContext commercePaymentMethodGroupRelsDispl
 ResultRow row = (ResultRow)request.getAttribute(WebKeys.SEARCH_CONTAINER_RESULT_ROW);
 
 CommercePaymentMethodGroupRel commercePaymentMethodGroupRel = (CommercePaymentMethodGroupRel)row.getObject();
+
+String engineKey = commercePaymentMethodGroupRel.getEngineKey();
+
+String modalId = "modal" + engineKey;
+
+int subscriptionEntryCount = commercePaymentMethodGroupRelsDisplayContext.getSubscriptionEntryCount(engineKey);
+int recurringCPDefinitionsCount = commercePaymentMethodGroupRelsDisplayContext.getRecurringCPDefinitionsCount(commercePaymentMethodGroupRel);
+
+boolean isLastRecurringPaymentMethod = commercePaymentMethodGroupRelsDisplayContext.isLastRecurringPaymentMethod(engineKey);
```

Comment:
variable should not start with "is" -- I guess just `lastRecurringPaymentMethod` works.